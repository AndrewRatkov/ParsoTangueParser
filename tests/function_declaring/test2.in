def int function(int a, int b)
return a + b;
enddef