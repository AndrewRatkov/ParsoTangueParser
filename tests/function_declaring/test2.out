[DEF FUN function(INTEGER a, INTEGER b)=>INT]
|   
+-->[RETURN]--->[INT,+ ]------->[INT,a]
                    |
                    +---------->[INT,b]
